troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -1,14 +1,15 @@
 
 define(
   [
-    'react',
+    'react/addons',
     'backbone',
     'jquery',
     'stores',
+    'modals',
     // plugins
-    'bootstrap'
+    // 'bootstrap' // Not used?
```

Comment:
I ran into the issue (I believe) in `Header.react`: 
```javascript
    // plugin: required to enable the drop-down, but not used directly
    bootstrap = require('bootstrap');
```
source: [blame: troposphere/static/js/components/Header.react.js](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/Header.react.js#L10-L11)

But I'm the commenter here saying "hey - this `require(...)` seems extraneous but it's not". I wish I remembered what the problem symptom was.